resources/views/account/partials/order/products.blade.php

```diff
@@ -16,9 +16,9 @@
                 <img
                     class="max-h-[100px] max-w-[150px]"
                     :alt="item.name"
-                    :src="'/storage/resizes/200/magento/catalog/product' + item.image"
+                    :src="`/storage/${$root.config.store}/resizes/200/sku/${item.sku}`"
```

Comment:
I think you can just use `config.store` as of: https://github.com/rapidez/core/blob/ffaa4c10b54b30787a166a6a8822d643e7a48196/resources/js/package.js#L27, but you've the test it